src/xia2/Applications/xia2setup.py

```diff
@@ -202,7 +202,11 @@ def visit(directory, files):
                     "Ignoring %s (Registry can not find format class)" % full_path
                 )
                 continue
-            elif format_class.ignore():
+            elif getattr(
+                format_class, "ignore", getattr(format_class, "is_abstract")
+            )():
+                # temporary workaround while not all format classes have the
+                # ignore method - https://github.com/xia2/xia2/issues/567
```

Comment:
Do we have any cases where xia2 will be encountering `is_ignore`, or classes without `is_abstract()`? Or just defensive check?
```suggestion
                # temporary workaround while not all format classes use the
                # is_abstract method - https://github.com/xia2/xia2/issues/567
```

Reply:
Not certain I agree - this gets the `ignore()` method first, and if that is `None` gets the `is_abstract()` method. 

Ah, because the code change is wrong -> fixing.